<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SignUpDialog</class>
 <widget class="QDialog" name="SignUpDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Sign up</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
</string>
  </property>
  <widget class="QPushButton" name="joinButton">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>320</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
	color:red;
	border: 1px solid #000000;
					 border-radius: 15px;
     background-color:rgba(255, 255, 255, 255);

}

QPushButton:pressed
{
	color:red;
     background-color:rgba(0, 145, 255, 128);
}</string>
   </property>
   <property name="text">
    <string>Join</string>
   </property>
  </widget>
  <widget class="QLabel" name="usernameLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
</string>
   </property>
   <property name="text">
    <string>username</string>
   </property>
  </widget>
  <widget class="QLabel" name="passwordLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
</string>
   </property>
   <property name="text">
    <string>password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="usernameBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue;
border: 1px solid #000000;
border-radius: 5px;
background-color:rgba(255, 255, 255, 255);
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue;
border: 1px solid #000000;
border-radius: 5px;
background-color:rgba(255, 255 255, 255);
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="cancelButton">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>320</y>
     <width>111</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
QPushButton
{
	border: 1px solid #000000;
					 border-radius: 15px;
     background-color:rgba(255, 255, 255, 255);

}

QPushButton:pressed
{
     background-color:rgba(0, 145, 255, 128);
}</string>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLabel" name="errorLabel">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>20</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:red;

</string>
   </property>
   <property name="text">
    <string>Error Message</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="groupIdLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>110</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
</string>
   </property>
   <property name="text">
    <string>group id</string>
   </property>
  </widget>
  <widget class="QLabel" name="groupPasswordLabel">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>160</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:black;
</string>
   </property>
   <property name="text">
    <string>group password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="groupPasswordBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>180</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue;
border: 1px solid #000000;
border-radius: 5px;
background-color:rgba(255, 255 255, 255);
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="groupIdBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>161</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:blue;
border: 1px solid #000000;
border-radius: 5px;
background-color:rgba(255, 255, 255, 255);
</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QRadioButton" name="createAdminRadio">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>270</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Create Admin Account</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="createUserRadio">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>240</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Create User Account</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
